packages/react-icons/src/utils/createFluentIcon.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from "react";
+import { FluentIconsProps } from "./FluentIconsProps.types";
+import wrapIcon from "./wrapIcon";
+
+export type FluentIcon = {
```

Comment:
Also reduce the size of `chunk-x.d.ts`. If not, every icon will produce very long and same type define. Prefer to use `type` instead of `interface` here because it's more transparent and avoid breaking changes in some extreme case.

Reply:
That's great improvement, it's what I also mentioned when was checking this area.

However, I think that it could be improved even more. In this PR we have:

```ts
export declare const Accessibility16Regular: import("../utils/createFluentIcon").FluentIcon;
export declare const Accessibility20Filled: import("../utils/createFluentIcon").FluentIcon;
export declare const Accessibility20Regular: import("../utils/createFluentIcon").FluentIcon;
````

But we can do following:

```ts
export declare const AccessibilityRegular: FluentIcon;
export declare const AccessibilityCheckmarkFilled: FluentIcon;
export declare const AccessibilityCheckmarkRegular: FluentIcon;
```

We need to do following changes to achieve that:

```diff
  for(const chunk of iconChunks) {
    chunk.unshift(`import { createFluentIcon } from "../utils/createFluentIcon";`);
+   chunk.unshift(`import type { FluentIcon } from "../utils/createFluentIcon";`);
  }
```

```diff
-      var jsCode = `export const ${destFilename} = /*#__PURE__*/createFluentIcon('${destFilename}', ${width}, [${paths}]);`
+      var jsCode = `export const ${destFilename}: FluentIcon = /*#__PURE__*/createFluentIcon('${destFilename}', ${width}, [${paths}]);`
```